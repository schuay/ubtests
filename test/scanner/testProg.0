(*test *)

if a = 3 then goto line 3 end;

label: a + +;

func (a, b, d); 