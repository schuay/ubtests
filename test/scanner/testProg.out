if 
ident a 
= 
num 3 
then 
goto 
ident line 
num 3 
end
;
ident label
: 
identa 
+ 
+
;
ident func 
(
ident a
, 
ident b
, 
ident d
)
; 